python/scripts/add_users_all_org_members_github_team.py

```diff
@@ -81,17 +82,16 @@ def run(github_service: GithubService, gql_client: Client):
 
 
 def main():
-    if len(sys.argv) == 2:
-        # Get the GH Action token
-        oauth_token = sys.argv[1]
-    else:
-        raise ValueError("Missing a script input parameter")
+    org_token = os.getenv("ADMIN_GITHUB_TOKEN")
+    if org_token == "" or org_token is None:
```

Comment:
I believe this could be
```
if not org_token
```

So both `""` and `None` return false 🤔

Reply:
Nice, will add it across the files in the next PR which will be the last script to port over, cheers